x/distribution/client/cli/tx.go

```diff
@@ -40,6 +45,28 @@ func GetTxCmd(storeKey string, cdc *amino.Codec) *cobra.Command {
 	return distTxCmd
 }
 
+func splitGenerateOrBroadcast(cliCtx context.CLIContext, txBldr authtxb.TxBuilder, msgs []sdk.Msg) error {
```

Comment:
Would you mind to write a small test case for this function?

Reply:
The reason I didn't include a test is because I am still not sure about the policy with respect to gaiacli tests now that there has been a split.
I can add something alike a unit test, no problem.